### PR TITLE
QRadar get mapping fields

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28162

## Description
Added get-mapping-fields command

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py [hunk 1]
```diff
@@ -529,6 +530,14 @@ def upload_indicators_list_request(self, reference_name, indicators_list):
             "POST", url, params=params, data=json.dumps(indicators_list)
         )
 
+    def get_custom_fields(
+            self, limit: Optional[int] = None) -> List[dict]:
+        url = urljoin(self._server, "api/config/event_sources/custom_properties/regex_properties")
+        headers = self._auth_headers
+        if limit:
```
Comment (Itay4) [inline Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py:537]: i dont see you pass `limit` to the function, so either do so or remove it

File: Packs/QRadar/Integrations/QRadar_v2/README.md [hunk 2]
```diff
@@ -1353,3 +1353,20 @@ There is no context output for this command.
 #### Human Readable Output
 
 >fetch-incidents was reset successfully.
+
+### get-mapping-fields
+***
+Returns the list of fields for an incident type. This command is for debugging purposes.
+This command is used in [Classification & Mapping](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/incidents/classification-and-mapping/create-a-mapper.html) section.
```
Comment (Itay4) [inline Packs/QRadar/Integrations/QRadar_v2/README.md:1360]: please move this line and the one below it to the `Fetch incidents` section in this README under the title `#### Field Mapping`
need to change it a bit also, to something like:
```
The integration support mapping fields based on schema.
The scheme is divided to 4 sections. Offense (root), Events: Builtins, Events: Custom Fields and Assets.
For more details, see the [Classification & Mapping documentation](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/incidents/classification-and-mapping/create-a-mapper.html).
```

File: Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py (hunk 1 above)
Comment (jochman) [reply Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py:537]: it's ok. It'll be used in the next enhancement ill open pr for today.

File: Packs/QRadar/Integrations/QRadar_v2/README.md (hunk 2 above)
Comment (jochman) [reply Packs/QRadar/Integrations/QRadar_v2/README.md]: DONE APPROVE PLEASE

File: Packs/QRadar/Integrations/QRadar_v2/README.md [hunk 3]
```diff
@@ -45,6 +47,10 @@ In order to use the integration with an API token you'll first need to change th
 To start fetching incidents, enable the parameter `Long running instance` - this will start a long running process that'll fetch incidents periodically.
 Depending on the system load, **the initial fetch might take a long time**.
 
+The integration support mapping fields based on schema.
```
Comment (Itay4) [inline Packs/QRadar/Integrations/QRadar_v2/README.md:50]:
```suggestion
#### Field (Schema) Mapping
The integration support mapping fields based on schema.
```

**Review by Itay4 — APPROVED** [review]

**Pull request merged**